src/util/rangePolygon.js

```diff
@@ -15,7 +15,7 @@ export default function rangePolygon (chart){
             return lbornrhi !== 'NA' ? chart.y(+lbornrhi) : 0
         });
 
-    var dRow = chart.filtered_data[0];
+    var dRow = chart.raw_data[0];
 
```

Comment:
Similarly, `chart.raw_data` refers to the multiple's data, which is already filtered on a single measure.